tox.ini

```diff
@@ -152,7 +156,7 @@ basepython = {[testenv:pypy3]basepython}
 deps =
     {[pinned]deps}
     PyPyDispatcher==2.1.0
-commands = {[testenv:pypy3]commands}
+commands = {[pinned]commands}
```

Comment:
This was probably OK before, `testenv:pypy3` above does not use `--doctest-modules`. I suspect this change may be responsible for the job failing now (maybe coverage breaks pypy?)

Reply:
I undid the change, but the test failed again. So I added a custom command for `pypy3-pinned`

Reply:
> I remember that one was failing

I did re-run the failing test in case it was a random issue, but I did not expect it to pass :sweat_smile: